File: ReSharper.FSharp/test/data/cache/csharpResolve/Methods 01.cs
using System.Collections.Generic;
using static Module;

public class Class1
{
  public Class1()
  {
    var t = new T();
    int m = t.Method();
    int sm = T.StaticMethod();

    int oInt = t.Overloads(123, 123);
    string oString = t.Overloads("123", "123");

    int oStaticInt = T.StaticOverloads(123, 123);
    string oStaticString = T.StaticOverloads("123", "123");
  }
}

File: ReSharper.FSharp/test/data/cache/csharpResolve/Methods 01.fs
module Module

type T() =
    member x.Method() = 123
    static member StaticMethod() = 123

    member x.Overloads(a, b) = a + b
    member x.Overloads(a: string, b: string) = a + b

    static member StaticOverloads(a, b) = a + b
    static member StaticOverloads(a: string, b: string) = a + b